freenas/cli/plugins/system.py

```diff
@@ -658,7 +658,13 @@ def __init__(self, name, context):
         self.add_property(
             descr='HTTPS certificate',
             name='https_certificate',
-            get='webui_https_certificate',
+            get=lambda o: get_related(self.context, 'crypto.certificate', o, 'webui_https_certificate'),
+            set=lambda o, v: set_related(self.context, 'crypto.certificate', o, 'webui_https_certificate', v),
+            usage=_("""\
+            Name of the certificate
+            """),
+            enum=lambda: self.context.entity_subscribers['crypto.certificate'].query(
```

Comment:
There's EntitySubscriberComplete for that:

<complete=EntitySubscriberComplete('https_certificate=', 'crypto.certificate', lambda o: o['name'] if o['type'] != 'CERT_CSR' else None),>